### PR TITLE
Fix SLES4SAP update test

### PR DESCRIPTION
**Pull request by dzedro**
- Verification run: http://10.100.12.155/tests/5590

### CODE REVIEW

File: lib/registration.pm [hunk 1]
```diff
@@ -431,7 +431,6 @@ sub fill_in_registration_data {
                 assert_screen 'yast-scc-emptypkg';
                 send_key 'alt-a';
             }
-            accept_addons_license('ha') if (check_var('SLE_PRODUCT', 'sles4sap'));
```
Comment (mimi1vx) [inline lib/registration.pm]: are you sure ? this isnt produduct bug ?

File: products/sle/main.pm [hunk 2]
```diff
@@ -333,6 +333,12 @@ if (is_updates_test_repo && !get_var('MAINT_TEST_REPO')) {
     set_var('ADDONURL',     '');
     set_var('ADDONURL_SDK', '');
 
+    # SLES4SAP does not have addon on SLE12SP3, is not present in SCC_ADDONS
+    if (is_sle('<15') && check_var('FLAVOR', 'Server-DVD-SLES4SAP-Updates')) {
+        $incidents{SLES4SAP} = get_var(uc($a) . '_TEST_ISSUES');
```
Comment (mimi1vx) [inline products/sle/main.pm]: `uc($a) `are you sure I think right is here `get_var('SLE4SAP_TEST_ISSUES','')`

or you can push `sle4sap` to `@addons` --> `push @addons 'sle4sap'`

File: lib/registration.pm (hunk 1 above)
Comment (dzedro) [reply lib/registration.pm]: not on SLE12SP3, there is no HA at all, but on SLE 15 maybe, there are multiple modules and not single license shown
[SLE12SP3](http://10.100.12.155/tests/5589#step/scc_registration/11)
[SLE15](https://openqa.suse.de/tests/1772887#step/scc_registration/9)

File: products/sle/main.pm (hunk 2 above)
Comment (dzedro) [reply products/sle/main.pm]: no i can't, then it will look for SLES4SAP module

Comment (dzedro) [reply products/sle/main.pm]: fixed

File: lib/registration.pm (hunk 1 above)
Comment (dzedro) [reply lib/registration.pm]: ah i just found the licenses https://openqa.suse.de/tests/1772887#step/accept_license/1

**Pull request merged**